CONTRIBUTING.md

```diff
@@ -97,49 +97,57 @@ to minimize diff review.
 
 ### Required dependencies
 
-#### Flutter
-
-[Install Flutter](https://flutter.dev/docs/get-started/install/linux) - the
-currently used version is specified in `.tool-versions`. If you're using
-[asdf](https://asdf-vm.com/) to manage your Flutter SDK, you can simply run
-`asdf install` to install the required version.
+[Install Flutter](https://flutter.dev/docs/get-started/install/linux) - the currently used version is specified in `.fvmrc`. If you're using [fvm](https://fvm.app) to manage your Flutter SDK, you can simply run `fvm install` to install the required version.
```

Comment:
Should we keep 80 (or 100 maybe?) max line length like before, so that it's easier to read in non-wrapping editors